src/main/java/fr/esgi/al5_2/Tayarim/repositories/AdministrateurRepository.java

```diff
@@ -0,0 +1,17 @@
+package fr.esgi.al5_2.Tayarim.repositories;
+
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import lombok.NonNull;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.Optional' import. Should be before 'org.springframework.stereotype.Repository'.